draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1456,6 +1442,45 @@ Subprotocol Definition.
 
 --- back
 
+# Updates to RFC7641
+
+## Notifications and Reordering
+
+When using the Observe Option with CoAP over UDP, notifications from the server set the option
+value to an increasing sequence number for reordering detection on the client since messages
+can arrive in a different order than they were sent. This sequence number is not required
+for CoAP over reliable transports since the TCP protocol ensures reliable and ordered delivery
+of messages. The value of the Observe Option in 2.xx notifications MAY be empty on transmission

```

Comment:
I suspect that the "MAY be empty" means that the Length field of the Observe option is 0.